Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+/*
+Run test from Jenkins build
+*/
+
+def project = 'murano_benchmark'
+pipeline {
+    options {
+        disableConcurrentBuilds()
+    }
+    agent any
+    triggers {
+        cron('H 1 * * *')
+    }
+    parameters {
+      string(name: 'domain', defaultValue: 'perftest.apps.dev1.murano.exosite-dev.com')
+    }
+    stages {
+        stage('test') {
+            steps {
+                sh "DOMAIN=${params.domain} > murano.env"
```

Comment:
docker compose actually supports variable substitution, https://docs.docker.com/compose/compose-file/#variable-substitution, but this is fine too